src/test/scala/org/broadinstitute/dsde/firecloud/EntityServiceSpec.scala

```diff
@@ -586,7 +622,7 @@ class EntityServiceSpec extends BaseServiceSpec with BeforeAndAfterEach {
   private def getEntityService(mockGoogleServicesDAO: MockGoogleServicesDAO = new MockGoogleServicesDAO,
                                mockImportServiceDAO: MockImportServiceDAO = new MockImportServiceDAO,
                                rawlsDAO: MockRawlsDAO = new MockRawlsDAO,
-                               cwdsDAO: MockCwdsDAO = new MockCwdsDAO) = {
+                               cwdsDAO: MockCwdsDAO = new MockCwdsDAO(false)) = {
```

Comment:
The tests that require `MockCwdsDAO` to be enabled specifically enable it, so by default it should be disabled.